Tree_Verticalorder_Traversal/Tree_Verticalorder_Traversal.java

```diff
@@ -0,0 +1,126 @@
+/* Vertical Order Traversal of Binary Tree is a traversal in which all nodes that fall in the same vertical line are visited together, starting from the leftmost vertical line and ending at the rightmost vertical line. The horizontal distance is measured with root serving as reference, then we measure the left and right deviations of the each node. */
+
+import java.util.*;
+
+// Node of Binary Tree storing data, horizontal distance from root, left and right child information
+class Node 
+{
+    int data;
+    int distance;
+    Node left, right;
+    Node(int val) 
+    {
+        data = val;
+        left = null;
+        right = null;
+    }
+}
+
+class Traversal
```

Comment:
Were you able to run this file with this class name?

Reply:
I was running in an online compiler, so it ran fine. However Class name and file name should be same to run the file from cmd. So I will make the required change.